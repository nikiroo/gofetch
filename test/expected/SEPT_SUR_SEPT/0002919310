   ARRÊTÉ APRÈS AVOIR PERDU UN PORTEFEUILLE REMPLI DE FAUX BILLETS 

  [Monde] 26-09-2018 16:32

  o News link: http://s.7s7.be/3479982
  o Source link: 


    Par: rédaction
    
    26/09/18 - 16h32 Source: Belga    © photo news.
    
    Un  homme  de  22  ans a été interpellé en Autriche après avoir
    perdu  son  portefeuille  dans  lequel  se trouvaient plusieurs
    faux  billets  de  50  euros,  a  annoncé  mercredi  la  police
    d'Eisenstadt (est).
    
    Identifié  grâce  à sa (vraie) carte d'identité qui se trouvait
    également  dans  le  portefeuille,  le  suspect a indiqué avoir
    acheté  les  faux  billets  sur  internet  afin  de  pouvoir se
    procurer de la drogue.
    
    La  police  autrichienne  avait annoncé mardi avoir contribué à
    l'identification  et  à l'arrestation en Pologne d'un faussaire
    soupçonné  d'avoir  écoulé  quelque  10.000  faux billets de 50
    euros via le Darknet, la partie obscure d'internet.



