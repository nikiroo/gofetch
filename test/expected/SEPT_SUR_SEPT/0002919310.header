0Arrêté après avoir perdu un portefeuille rempli de faux billets	null/SEPT_SUR_SEPT/0002919310		70
i  [Monde] 26-09-2018 16:32
i
i    Un  homme  de  22  ans a été interpellé en Autriche après avoir
i    perdu  son  portefeuille  dans  lequel  se trouvaient plusieurs
i    faux billets de 50 euros, ...
i
