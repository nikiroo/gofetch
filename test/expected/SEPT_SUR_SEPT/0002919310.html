<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h1>Arrêté après avoir perdu un portefeuille rempli de faux billets</h1>
	<div class='details'>([Monde] 26-09-2018 16:32)</div>
	<br/>
    <ul>
        <li>News link: <a href='http://s.7s7.be/3479982'>http://s.7s7.be/3479982</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Par: rédaction<br/><br/>26/09/18 - 16h32 Source: Belga    © photo news.<br/><br/>Un homme de 22 ans a été interpellé en Autriche après avoir perdu son portefeuille dans lequel se trouvaient plusieurs faux billets de 50 euros, a annoncé mercredi la police d&#x27;Eisenstadt (est).<br/><br/>Identifié grâce à sa (vraie) carte d&#x27;identité qui se trouvait également dans le portefeuille, le suspect a indiqué avoir acheté les faux billets sur internet afin de pouvoir se procurer de la drogue.<br/><br/>La police autrichienne avait annoncé mardi avoir contribué à l&#x27;identification et à l&#x27;arrestation en Pologne d&#x27;un faussaire soupçonné d&#x27;avoir écoulé quelque 10.000 faux billets de 50 euros via le Darknet, la partie obscure d&#x27;internet.
	</div>
<hr/>
</div>
</body>
